occupancy_threshold: 99
planning_rate: 10
publish_viz_paths: false
publish_path_names: true
print_timings: false
line_check_resolution: 0.1
pixels_to_skip: 0
open_cv_scale: 0.5
h_class_threshold: 0.01
min_node_sep_sq: 1.0
publish_all_path_markers: true
visualize_edges: true
node_connection_threshold_pix: 1.0
collision_threshold: 90
trimming_collision_threshold: 60
search_radius: 1.5
lonely_branch_dist_threshold: 4
path_waypoint_sep: 0.1
publish_path_point_markers: false
path_vertex_angle_threshold: 45
path_vertex_dist_threshold: 0.8
inflation_radius: 0.5
inflation_blur_radius: 0.125
base_link_frame: base_link
debug_path_id: false
robot_radius: 0.3 #Param used by eband optimization
use_elastic_band: false
backtrack_plan_threshold: 5 #Plan max length in meters

num_optim_iterations: 3
max_recursion_depth_approx_equi: 4
internal_force_gain: 1.0
external_force_gain: 2.0
tiny_bubble_distance: 0.01
tiny_bubble_expansion: 0.01
min_bubble_overlap: 0.7
equilibrium_relative_overshoot: 0.75
significant_force: 0.15
costmap_weight: 10.0
